sfdx-project.schema.json

```diff
@@ -1,412 +1,146 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
there's a lot of diffs for this file.  The valid/invalid tests are passing so it's probably reasonably close.